### PR TITLE
 List all Security Groups

### PR DESCRIPTION
**Pull request by s-bortolussi**
This change adds the API to list all security groups (GET /v2/security_groups)

[#101522660][#512]

### CODE REVIEW

File: cloudfoundry-client/src/main/java/org/cloudfoundry/client/v2/securitygroups/_ListSecurityGroupsRequest.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+     * The names
+     */
+    @InFilterParameter("name")
+    @Nullable
+    abstract List<String> getNames();
+
+    /**
+     * The running default
+     */
+    @IsFilterParameter("running_default")
```
Comment (twoseat) [inline cloudfoundry-client/src/main/java/org/cloudfoundry/client/v2/securitygroups/_ListSecurityGroupsRequest.java]: Hi @s-bortolussi - I don't believe this (or the staging_default equivalent) are legitimate filters. Can you point me to the doc that makes you think they are?

Comment (s-bortolussi) [inline cloudfoundry-client/src/main/java/org/cloudfoundry/client/v2/securitygroups/_ListSecurityGroupsRequest.java]: Hi @twoseat - You are right. The [doc](https://apidocs.cloudfoundry.org/latest-release/security_groups/list_all_security_groups.html) only says that 'name' is a valid filter.
I will remove invalid filters. Sorry about that.

Comment (twoseat) [inline cloudfoundry-client/src/main/java/org/cloudfoundry/client/v2/securitygroups/_ListSecurityGroupsRequest.java]: That's ok, I'll pick it up on a polishing commit - I just wanted to make sure I hadn't missed something. Thanks!

**Comment by s-bortolussi** [comment]
Invalid filters removed

**Comment by twoseat** [comment]
Closing - merged but missed the PR reference in the merge.

**Pull request closed without merging**